<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ALIPL0617" ignored="0" name="linkedinTest" tests="3" failures="1" timestamp="18 Feb 2019 09:26:51 GMT" time="16.73" errors="0">
  <testcase name="@AfterMethod ProgramTermination" time="0.0" classname="learnautomation.linkedin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at learnautomation.linkedin.ProgramTermination(linkedin.java:38)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod ProgramTermination -->
  <testcase name="@AfterMethod ProgramTermination" time="0.0" classname="learnautomation.linkedin">
    <skipped/>
  </testcase> <!-- @AfterMethod ProgramTermination -->
  <testcase name="@AfterMethod ProgramTermination" time="0.0" classname="learnautomation.linkedin">
    <skipped/>
  </testcase> <!-- @AfterMethod ProgramTermination -->
  <testcase name="loginToLinkedin" time="16.668" classname="learnautomation.linkedin">
    <failure type="java.lang.AssertionError" message="Invalid credentials expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Invalid credentials expected [true] but found [false]
at learnautomation.linkedin.loginToLinkedin(linkedin.java:31)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToLinkedin -->
  <testcase name="loginToLinkedin" time="0.0" classname="learnautomation.linkedin">
    <skipped/>
  </testcase> <!-- loginToLinkedin -->
  <testcase name="loginToLinkedin" time="0.0" classname="learnautomation.linkedin">
    <skipped/>
  </testcase> <!-- loginToLinkedin -->
</testsuite> <!-- linkedinTest -->
